src/Pmi/Entities/Participant.php

```diff
@@ -383,20 +383,28 @@ private function getSiteSuffix($site)
 
     private function getActivityStatus($participant)
     {
+        // Withdrawn
         if (in_array($participant->withdrawalStatus, self::$withdrawalStatusValues, true)) {
             return 'withdrawn';
-        } elseif (in_array($participant->deceasedStatus, self::$deceasedStatusValues, true)) {
-            return 'deceased';
-        } else {
-            switch (isset($participant->suspensionStatus) ? $participant->suspensionStatus : null) {
+        }
+
+        // Deactivated
+        if (isset($participant->suspensionStatus)) {
+            switch ($participant->suspensionStatus) {
                 case 'NOT_SUSPENDED':
                     return 'active';
```

Comment:
I think in the case of suspensionStatus = NOT_SUSPENDED, we then want to check for the deceased status. I think suspension status should always be NO_CONTACT or NOT_SUSPENDED. We can confirm with the RDR.

I believe what we want is just:

```php
if (isset($participant->suspensionStatus) && $participant->suspensionStatus === 'NO_CONTACT')) {
    return 'deactivated';
}
```
 Then, check for deceased. And then return active if none apply.

We should check to see if an empty value should ever be returned.

Reply:
🐼 I'd forgotten that these were not mutually exclusive statuses.

Reply:
Resolved in f75a22d3